hydra-test-utils/hydra-test-utils.cabal

```diff
@@ -0,0 +1,49 @@
+cabal-version: 2.2
+
+name: hydra-test-utils
+version: 1.0.0
+description: Hydra utilities for testing, aka. "Test Prelude"
+author: IOG
+copyright: 2021 IOG
+license: Apache-2.0
+license-files:
+  LICENSE
+  NOTICE
+build-type: Simple
+extra-source-files: README.md
+
+source-repository head
+  type: git
+  location: https://github.com/input-output-hk/hydra-poc
+
+library
+  hs-source-dirs:
+    src
+  exposed-modules:
+    Hydra.Test.Prelude
+    Hydra.Network.Ports
```

Comment:
Could: I would expect all modules of a package called "test-utils" to include some `Test` in the module name. Furthermore, other packages used in tests typically start with `Test` (e.g. `Test.QuickCheck`) and maybe we could name the prelude then also `Test.Hydra.Prelude`?

Reply:
Good point. I will rename the `Ports` module. I am fine with`Test.Hydra.XXX` prefix, will do that renaming too.